tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,28 @@
+export const GOLF_SCORES = {
+  HOLE_IN_ONE: 'Hole-in-one!',
+  EAGLE: 'Eagle',
+  BIRDIE: 'Birdie',
+  PAR: 'Par',
+  BOGEY: 'Bogey',
+  DOUBLE_BOGEY: 'Double Bogey',
+  GO_HOME: 'Go Home!',
+};
+
+export const golfScore = (par, stroke) => {
+  switch (true) {
```

Comment:
it is not very good way when you pass true and then have some conditions
Try to rewrite it to `stroke - par` the result will be difference, cut edge case when it -2 or smaller(-3...) then you will have in switch just numbers `-1, 0, 1, 2` and default when it is 3 or bigger.
when stroke is equal 1 you can cut in first line before doing calculations